### ISSUE TITLE
Option to keep margins for black pages

### ISSUE THREAD

**Issue by PringlePot** [ciromattia/kcc#583]
A lot of manga have black margins on the pages to show that its a memory / happening in the past. KCC however gets rid of the margins which makes it sometimes confusing when reading.

**Comment by darodi** [comment]
You can use `cropping mode` and `cropping power`
![image](https://github.com/ciromattia/kcc/assets/4682830/a521f7e6-5ffd-4c53-b5de-c886128e2442)

**Comment by PringlePot** [comment]
So disabling would disable cropping? It's kinda vague. The cropping is great for white pages

**Comment by axu2** [comment]
I think you can just use the preserve margin % option for this. @PringlePot 

So like preserve 10% of margins or something

I suppose we could set a minimum margin preservation % for black backgrounds but not sure what that % should be, I would defer to you for a specific number.

**Comment by axu2** [comment]
@nitram3 Do you think black background pages should preserve the background by setting a default "preserve margin" % like 10%?

If so, what % would you prefer?

- related #1350 

<img width="233" height="78" alt="Image" src="https://github.com/user-attachments/assets/44356f94-bff2-4267-883b-4776c5c6b3cd" />

**Comment by nitram3** [comment]
So, a "preserve margin" % for only Black BG pages, but not for White ones?

I have Kindle Paperwhite 6 and my father has a Paperwhite 4. Mine is wider than his. So, we set any comic on KCC for my kindle, and without margins. Then on KOReader, we set "**full fit**" and **no page crop**. The result is as much zoomed in pages as we can get for any device.
Works like a charm.
Except for the black BG pages. KOReader doesn't fill the background with black / dark color. So, it's an imperfect solution.
However if I was only exporting comics for myself, I would use W/B margins. KCC fills the left/right side with the correct color based on the page. That seems to me like the perfect output.

TLDR; I don't think a default "preserve margin" % is needed for dark pages. But if you do decide to go forward with the implementation; my opinion is that separate control for *Top/Bottom* and *Left/Right* margins is needed.

**Comment by axu2** [comment]
> KOReader doesn't fill the background with black / dark color.

did not know this, most readers do, maybe feature request koreader.

But overall, I'll think this over
